### PR TITLE
Added padding to popup form

### PR DESCRIPTION
**Pull request by LaurenChoquer**

### CODE REVIEW

**Comment by BradenMccoy** [comment]
This pull request adds additional space on the final Exporter Setup popup to improve it's appearance.

File: exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/Wizard/Pages/ExportOrAdvancedForm.cs [hunk 1]
```diff
@@ -33,5 +33,10 @@ private void exportRobotButton_Click(object sender, EventArgs e)
             Close();
             StandardAddInServer.Instance.ForceExport();
         }
+
+        private void MainLayout_Paint(object sender, PaintEventArgs e)
```
Comment (HiceS) [inline exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/Wizard/Pages/ExportOrAdvancedForm.cs]: Why is this empty?

Comment (matthew17754) [reply exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/Wizard/Pages/ExportOrAdvancedForm.cs]: I noticed that too. Those are created as event handlers when double-clicking a component in the WinForm editor. They are often auto-generated by accident and the best way I can think of to remove them is to simply delete the handler and remove it's reference from Designer.cs

Comment (LaurenChoquer) [reply exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/Wizard/Pages/ExportOrAdvancedForm.cs]: I can make that change and push again to this branch.

**Comment by HiceS** [comment]
thank you 👍

**Pull request merged**